### DISCUSSION TITLE
Installation Issue: Missing category_types

### DISCUSSION THREAD

**Discussion by ssjohl** [balajidharma/basic-laravel-admin-panel#39] [General]
The direct installation has a database model issue with installation steps.
The step `php artisan migrate --seed --seeder=AdminCoreSeeder` fail complaining of a missing `category_types` base table.

**Comment by balajidharma** [comment]
I hope you following "[Without Docker Desktop](https://github.com/balajidharma/basic-laravel-admin-panel#without-docker-desktop)" 
try seeder after the below step 

`php artisan vendor:publish --provider="BalajiDharma\LaravelCategory\CategoryServiceProvider"`

I have updated the same in the [README.md](https://github.com/balajidharma/basic-laravel-admin-panel#readme)
